public/js/bundle.js

```diff
@@ -148,6 +151,20 @@ class TableView {
 		this.sheetBodyEl.appendChild(fragment);
 	}
 
+	renderTableFooter() {
```

Comment:
Ah this file should not have been checked in. It's a generated file, and generated artifacts should not be committed.  It clutters the codebase and every pull requests, as you can see here -- all the same code is showing up again.  

Please confer with Kay on how to remove it from git tracking and make sure it's added to your `.gitignore` file.